### DISCUSSION TITLE
[Feature request] A way to configure the start and end time in wavesurfer

### DISCUSSION THREAD

**Discussion by amitdhiman000** [katspaugh/wavesurfer.js#3826] [Q&A]
I have a use case where, I want to draw waveform and play the audio from given start <-> end time.
I checked the [options](https://wavesurfer.xyz/docs/types/wavesurfer.WaveSurferOptions), but couldn't find anything helpful, although there is a way to configure the the duration but not helpful.

![wavesurfer](https://github.com/user-attachments/assets/e5924010-1a6b-458c-ac0b-f2df90d1b090)

Please suggest if there is a way to achieve it using wavesurfer or how this can be taken as a new feature request.

Thanks.

**Comment by psavva** [comment]
Maybe the regions plugin in a solution
